workflow-templates/im-run-tf-import.yml

```diff
@@ -0,0 +1,119 @@
+# Workflow Code: DrearyBuck_v1    DO NOT REMOVE
+
+# TODO: If the terraform is still using on-prem-egress-ips, it needs to be switched to network-information.
+# TODO: Ensure each of the repo-level and env-level secrets used in this workflow have been populated by an admin in your repository.
+
+name: Import Terrafrom State
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: The environment the import should be done in - dev|qa|stage|prod|demo|uat. # TODO:  Update for the environments that are available
+        required: true
+      branch-tag-sha:
+        description: The branch, tag or sha of the terraform that has the configuration for the resource(s) to import.
+        required: true
+      tf-resource-name:
+        description: The name of the resource to import.
+        requierd: false
+      tf-object-id:
+        description: The id of the object to import.
```

Comment:
Do you think you could give examples of these?  I can see this being a point of confusion for what exactly needs to be added to both inputs.

Reply:
I think I was trying too hard to match the Octopus step template to be honest. It was very confusing and I have no clue why things were named the way they were. I've updated the names to more closely match terraform's documentation and also added what I hope are helpful examples.

Reply:
I think those examples will really help.  It looks awesome!